src/components/timeline/Markers.js

```diff
@@ -377,10 +376,12 @@ const markersOptions: ExplicitConnectOptions<OwnProps, StateProps, {||}> = {
     const selectedThread = getSelectedThreadIndex(state);
     const markers = selectors.getCommittedRangeFilteredMarkersForHeader(state);
     return {
+      additionalClassName:
+        selectors.getThread(state).name === 'GeckoMain'
+          ? 'timelineMarkersGeckoMain'
+          : null,
```

Comment:
I find it a bit hacky to pass a class name here. The reason is that a class name is a rendering decision, rather than some logic.
My preference would be to pass the thread name itself to the component, and that the `render` method decides about the additional class name.

Reply:
I'm going to skip on this one. I'm thinking more in terms of each connect function configuring the more generic version. This seems good enough for now.

Reply:
>  I'm thinking more in terms of each connect function configuring the more generic version. 

I actually find that passing the thread name is more generic (as the difference is not necessarily just a class name).

But not a big deal.